app/app.go

```diff
@@ -83,17 +85,18 @@ import (
 	tmos "github.com/tendermint/tendermint/libs/os"
 	dbm "github.com/tendermint/tm-db"
 
+	"github.com/CosmWasm/wasmd/x/wasm"
```

Comment:
I think this is a remnant of the code from Juno, but chihuahua doesn't use cosmwasm, so we need to remove this